jme3-examples/src/main/java/jme3test/texture/TestTexture3D.java

```diff
@@ -116,16 +116,17 @@ public void simpleInitApp() {
          */
     private Texture getTexture() throws IOException {
         ArrayList<ByteBuffer> data = new ArrayList<ByteBuffer>(1);
-        ByteBuffer bb = BufferUtils.createByteBuffer(10 * 10 * 10 * 3);//all data must be inside one buffer
+        ByteBuffer bb = BufferUtils.createByteBuffer(10 * 10 * 10 * 4);//all data must be inside one buffer
         for (int i = 0; i < 10; ++i) {
             for (int j = 0; j < 10 * 10; ++j) {
                 bb.put((byte) (255f*i/10f));
                 bb.put((byte) (255f*i/10f));
                 bb.put((byte) (255f));
+                bb.put((byte) (255f));
             }
         }
         bb.rewind();
         data.add(bb);
-        return new Texture3D(new Image(Format.RGB8, 10, 10, 10, data, null, ColorSpace.Linear));
+        return new Texture3D(new Image(Format.RGBA8, 10, 10, 10, data, null, ColorSpace.Linear));
```

Comment:
Why?

Reply:
3D textures not having alpha channel weren't rendered at all

Reply:
Do you know why? We might need to investigate that...

Reply:
At first I didn't know, but I've just checked RGB8 and RGBA8 and they both work properly now. Could be related with latest changes I did to GLImageFormats to correct it on gles platforms. If you want, this can be reverted to RGB

Reply:
Yes, if possible revert also the changes in TestTextureArray.java, please.

Reply:
Just comited